ui/cypress/e2e/multipleCohorts.cy.js

```diff
@@ -40,17 +40,19 @@ describe("Basic tests", () => {
     cy.wait(2000);
 
     cy.iframe().find(`button[name='${featureSetName}']`).click();
-    cy.iframe().find("button:Contains(Data)").click();
 
     cy.iframe().find(`button[name='${cohortName1}']`).click();
+    cy.iframe().find("button:Contains(Data)").click();
```

Comment:
Just curious, what part of the PR prompted these test changes?

Reply:
The code fetching the preview entities in export.tsx wasn't previously refetching based on cohort changes. The new, shared version more correctly does since it's a parameter to the call.